app/src/ui/toolbar/push-pull-button.tsx

```diff
@@ -16,7 +16,7 @@ interface IPushPullButtonProps {
   readonly aheadBehind: IAheadBehind | null
 
   /** The name of the remote. */
-  readonly remoteName: string | null
```

Comment:
This is also a rename that we don't want to do. We know the remote name once the repository is cloned to disk, and can be configured by the user - we should display that in the app.